docs/roadmap.md

```diff
@@ -7,7 +7,7 @@ The list is based on the survey results and the discussion in wg-etcd-operator.
   - Users should be able to set at least the etcd version and cluster size
 - Understand health of a cluster
 - Enabling TLS communication
-  - Should also support certificate renewal
```

Comment:
why change this?

Reply:
I guess it got changed during rebase, fixing it